src/components/backdrop/backdrop.js

```diff
@@ -66,6 +71,13 @@ angular
         }
       });
 
+      function resize() {
```

Comment:
*  should we not also check and adjust the `width` ? 
*  what about *margins* ?